sql/planbuilder/dml.go

```diff
@@ -97,8 +117,13 @@ func (b *Builder) buildInsert(inScope *scope, i *ast.Insert) (outScope *scope) {
 			if len(srcScope.cols) == len(destScope.cols) {
 				combinedScope.newColumn(srcScope.cols[i])
 			} else {
-				// check for VALUES refs
-				c.table = OnDupValuesPrefix
+				// The to-be-inserted values can be referenced via the provided alias.
+				c.table = b.insertTableAlias
+				if len(b.insertColumnAliases) > 0 {
+					aliasColumnName := b.insertColumnAliases[c.col]
```

Comment:
strings to lower?

Reply:
Lowered the string on insert into the map.